themes/default/data/registry/packages/aws-iam.yaml

```diff
@@ -0,0 +1,14 @@
+category: Cloud
+component: true
+description: ""
+featured: false
+logo_url: ""
+name: aws-iam
+native: true
```

Comment:
This shouldn't be a native - you are missing keywords - see aws-apigateway for the ones you want

Reply:
@stack72 update the `native` value but not sure what else you're referring to. I compared it to the aws-apigateway data file and they look similar now.